feat: custom config fix, example custom config extended

irubido opened the pull request:
- fixed assertion bug for custom config, missing unit decimals
- parachains using polkadots bip345 with ss58 prefix  
- extended example functionality, input fields support custom config

BeroBurny reviewed (approved):
LGTM

mpetrunic reviewed (commented):
This is breaking change right?

irubido commented:
> This is breaking change right?

Not breaking any functionality from before, key derivation for polkadot, westend and kusama are all the same as they were.

Just fixed configure method bug , custom configurations were blocked by assert function not having all object properties.

And for creation of keyring of these substrate chains we use polkadot bip44 and address prefix provided in custom config.

And added Custom option in example
